include/seqan3/range/hash.hpp

```diff
@@ -24,7 +26,32 @@ namespace std
  */
 template <ranges::input_range urng_t>
     //!\cond
-    requires seqan3::semialphabet<seqan3::reference_t<urng_t>>
+    requires (!seqan3::const_iterable_range<urng_t>) && seqan3::semialphabet<std::ranges::range_reference_t<urng_t>>
```

Comment:
I remember that we discussed this and the overload was unnecessary or something?